.gitignore

```diff
@@ -7,4 +7,17 @@ node_modules
 stylecop.cache
 
 # CSHTML files should be generated
-*cshtml
\ No newline at end of file
+*cshtml
+src/cordova/www/images/v1/images/logo2b.png
```

Comment:
There shouldn't be any images under src/cordova/www/images/v1, all images should be under src/cordova/www/images/*.*. So instead of this entry, can you add src/cordova/www/images/v1/**/* if you're getting them accidentally copied under v1 folder (not sure why?)